--- linux-3.18.36/arch/x86/include/asm/dmi.h	2016-06-22 23:02:29.000000000 -0500
+++ dmi.h	2016-07-12 12:38:51.815612205 -0500
@@ -15,7 +15,7 @@
 /* Use early IO mappings for DMI because it's initialized early */
 #define dmi_early_remap		early_ioremap
 #define dmi_early_unmap		early_iounmap
-#define dmi_remap		ioremap
+#define dmi_remap		ioremap_cache
 #define dmi_unmap		iounmap
 
 #endif /* _ASM_X86_DMI_H */
